README.md

```diff
@@ -1,158 +1,91 @@
-
-Kyverno policies are Kubernetes resources that can be written in YAML or JSON. Kyverno policies can validate, mutate, and generate any Kubernetes resources.
-
-Kyverno runs as a [dynamic admission controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) in a Kubernetes cluster. Kyverno receives validating and mutating admission webhook HTTP callbacks from the kube-apiserver and applies matching policies to return results that enforce admission policies or reject requests.
-
-Kyverno policies can match resources using the resource kind, name, and label selectors. Wildcards are supported in names.
-
-Mutating policies can be written as overlays (similar to [Kustomize](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays)) or as a [JSON Patch](http://jsonpatch.com/). Validating policies also use an overlay style syntax, with support for pattern matching and conditional (if-then-else) processing.
-
-Policy enforcement is captured using Kubernetes events. Kyverno also reports policy violations for existing resources.
-
-**NOTE** : Your Kubernetes cluster version must be above v1.14 which adds webook timeouts. To check the version, enter `kubectl version`.
+Kyverno is a policy engine built for Kubernetes:
+* policies as Kubernetes resources (no new language to learn!)
+* validate, mutate, or generate any resource
+* match resources using label selectors and wildcards
+* validate and mutate using overlays (like Kustomize!)
+* generate and synchronize defaults across namespaces
+* block or report violations 
+* test using kubectl 
  
 ## Quick Start
 
+**NOTE** : Your Kubernetes cluster version must be above v1.14 which adds webook timeouts. To check the version, enter `kubectl version`.
+
 Install Kyverno:
 ```console
 kubectl create -f https://github.com/nirmata/kyverno/raw/master/definitions/install.yaml
 ```
 
-You can also install using the [Helm chart](https://github.com/nirmata/kyverno/blob/master/documentation/installation.md#install-kyverno-using-helm).  As a next step, import [sample policies](https://github.com/nirmata/kyverno/blob/master/samples/README.md) and learn about [writing policies](https://github.com/nirmata/kyverno/blob/master/documentation/writing-policies.md). You can test policies using the [Kyverno cli](https://github.com/nirmata/kyverno/blob/master/documentation/kyverno-cli.md). See [docs](https://github.com/nirmata/kyverno/#documentation) for more details.
- 
-## Examples
-
-### 1. Validating resources
+You can also install Kyverno using a [Helm chart](https://github.com/nirmata/kyverno/blob/master/documentation/installation.md#install-kyverno-using-helm).
 
-This policy requires that all pods have CPU and memory resource requests and limits:
+Add the policy below. It requires that all pods have a `app.kubernetes.io/name` label:
```

Comment:
Can we briefly describe what this policy does? For those who are not familiar with Kyverno, they would know this is a validate policy to block resource creation.

Reply:
I added a few lines...